exercise-analyst-server/API/Services/MeasurementsDev/MeasurementsDevService.cs

```diff
@@ -20,16 +20,38 @@ public MeasurementsDevService(DataContext context, IMapper mapper)
             _mapper = mapper;
         }
 
-        public async Task<Response> CreateMeasurementAsync(CreateMeasurementDevRequest request)
+        public async Task<Response<int>> CreateMeasurementAsync(CreateMeasurementDevRequest request)
         {
             var measurement = _mapper.Map<Measurement>(request);
 
             _context.Add(measurement);
             var result = await _context.SaveChangesAsync();
 
             if(result > 0)
-                return new Response(HttpStatusCode.NoContent);
-            return new Response(HttpStatusCode.BadRequest, new[] { "No data has been saved" });
+                return new Response<int>(){
+                    HttpStatusCode = HttpStatusCode.Created,
+                    Payload = measurement.Id
+                };
+            return new Response<int>(){
+                HttpStatusCode = HttpStatusCode.BadRequest,
+                Errors = new[] { "No data has been saved" }
+            };
+        }
+
+        public async Task<Response<Measurement>> GetMeasurementAsync(int id)
```

Comment:
A tutaj nie powinniśmy używać Dto zamiast zwracać encję?

Reply:
To będzie usunięte ze względu na to co dostarcza @JJMPdsk  tlyko muszę przeciścić i na tym beanchu zostanie samo wersjonowanie api